example/config/deploy.rb

```diff
@@ -4,3 +4,4 @@
 set :application, 'my_app_name'
 set :repo_url, 'git@github.com:MindscapeHQ/capistrano-raygun-deployment.git'
 set :deploy_to, '/var/www/my_app_name'
+set :raygun_deployment_environments, [:production]
```

Comment:
Have you tested any deploy with that? I didn't yet, because the last problem broke of of my builds, I need to do a Isolated test.